### ISSUE TITLE
Training curve for RTS

### ISSUE THREAD

**Issue by chenshihfang** [visionml/pytracking#354]
Hi, thanks for your RTS work.

I found that the loss for initial filter iteration occasionally overfitting when training. (~110 epochs)
Is this usual?
![image](https://user-images.githubusercontent.com/113094161/191708155-2d12e421-3902-4b8e-918f-cce46ae61fad.png)
Due to the big GPU memory requirement for RTS,
so I change to batch size to 5 for training, 
but I think reducing the batch may not lead to network overfitting.

BTW, is it OK to reduce memory Instance Memory / Segmentation Memory sizes for reducing GPU memory when training? I guess the high GPU memory occupied may be because of this memory mechanism.

**Comment by mattpfr** [comment]
@sfchen94 Thanks for your comment. I would say that a batch size of 5 is pretty small. I understand that the memory requirement is quite high for training, but I would recommend to train on several GPUs to keep a bigger batch size. Training with a lower batch size might mean that you could have to adjust some other training parameters.
